### PR TITLE
feat: add paytm to create event and order-form page

### LINKED ISSUE #3388: Add PayTM as a payment option on order checkout page
**Is your feature request related to a problem? Please describe.**

As we are moving forward with PayTM on backend, we should start with FE integration

**Describe the solution you'd like**

- Add PayTM on event creation page as a payment method
- Add PayTM on order pending page (if paytm is selected as a payment method)

**Additional context**

On it

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #3388 

#### Changes proposed in this pull request:

- It adds paytm to create event and order-form page

_Create Event_
Screenshot_2019-08-13 Basic Details Edit Event Paytm Events Open Event

_New Order_
Screenshot_2019-08-13 New Order -a5185ec5-02ab-424a-9044-48d0edf25d37 Orders Open Event

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/models/setting.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ export default ModelBase.extend({
   stripeTestPublishableKey   : attr('string'),
   isAliPayActivated          : attr('boolean'),
   isPaypalActivated          : attr('boolean'),
+  isPaytmActivated           : attr('boolean'),
```
Comment (kushthedude) [inline app/models/setting.js]: Duplicate field

Comment (shreyanshdwivedi) [reply app/models/setting.js]: Will fix this asap

**Comment by shreyanshdwivedi** [comment]
@kushthedude as discussed with @uds5501 @mrsaicharan1 we are starting by accepting payment to current admin's account to see if everything works initially.
Then we'll shift to accept organizer's account and connect it later. I hope this makes sense

**Comment by mrsaicharan1** [comment]
> It's incomplete as soon as organiser select `Accept By PayTM`, A field should or button should pop-up to connect with PayTM, As done in Stripe.

Every payment gateway has its own unique workflow. The other implementation details can be looked at afterwards as this is a sub issue. This is fine.

**Comment by uds5501** [comment]
@shreyanshdwivedi Please take a look at Travis?

**Comment by shreyanshdwivedi** [comment]
@uds5501 @mrsaicharan1 @kushthedude @CosmicCoder96  it was a random fail. Please have a look

**Pull request merged**